Lib/glyphsLib/builder/features.py

```diff
@@ -630,6 +630,21 @@ def _process_feature_block(self):
             #    the block is only made of comments
         else:
             feature.code = self._rstrip_newlines(self.doc.text(contents))
+        if feature.automatic:
+            # See if there is a feature name in the code that should be
+            # written to the notes
+            feature_name = re.search(
+                r"(featureNames\s*{\s*name \"(.+)\";\s*};)",
+                feature.code,
+                flags=re.MULTILINE + re.DOTALL,
+            )
+            if feature_name:
+                statement, name = feature_name.groups()
+                if not re.search("^Name: (.+)", feature.notes):
+                    # Don't add if the name is already there
+                    feature.notes = f"Name: {name}\n" + feature.notes
```

Comment:
The feature names may go out of sync here, because we don't check if what's already in the feature notes contains the same name as the featureNames block. Do you think that's a problem?